docs/reference-guides/data/data-core-editor.md

```diff
@@ -248,6 +248,14 @@ _Returns_
 
 Returns the post type of the post currently being edited.
 
+_Usage_
+
+Get the current post type
+
+```js
+const currentPostType = wp.data.select('core/editor').getCurrentPostType()
```

Comment:
```suggestion
const currentPostType = wp.data.select( 'core/editor' ).getCurrentPostType()
```

Reply:
Looks like the whitespace is different than WP coding standards and that's what is failing .

Reply:
I got my formatting conflict fix up in vscode, and it appears to be using WP standards (it's been a min, been uring psr for a long time) but it's still failing. If I try and run `npm run lint:js:fix` I get a warning that stops the process, which also happens to me when I update trunk. This might be a new issue? I'll have to check the issues
 
<img width="1062" alt="Screenshot 2024-07-25 at 8 45 02 PM" src="https://github.com/user-attachments/assets/eec77d1c-525f-4c11-b06e-5778795270e6">